solutions/fully-configurable/variables.tf

```diff
@@ -0,0 +1,366 @@
+  description = "The `reply_to` email address used in any Security and Compliance Center events coming via Event Notifications."
+  default     = "no-reply@ibm.com"
+}
+
+variable "event_notifications_email_list" {
+  type        = list(string)
+  description = "The list of email addresses to notify when Security and Compliance Center triggers an event."
+  default     = []
+}
+
+##############################################################################
+# Logs Routing
+##############################################################################
+
+variable "logs_routing_tenant_regions" {
+  type        = list(any)
+  default     = []
+  description = "Pass a list of regions to create a tenant for that is targetted to the Cloud Logs instance created by this module. To manage platform logs that are generated by IBM Cloud® services in a region of IBM Cloud, you must create a tenant in each region that you operate. Leave the list empty if you don't want to create any tenants. NOTE: You can only have 1 tenant per region in an account."
+  nullable    = false
+}
+
+variable "skip_logs_routing_auth_policy" {
+  description = "Whether to create an IAM authorization policy that permits the Logs Routing server 'Sender' access to the IBM Cloud Logs instance created by this module."
+  type        = bool
+  default     = false
+}
+
+#############################################################################################################
+# Logs Policies Configuration
+#
+# logs_policy_name -The name of the IBM Cloud Logs policy to create.
+# logs_policy_description - Description of the IBM Cloud Logs policy to create.
+# logs_policy_priority - Select priority to determine the pipeline for the logs. High (priority value) sent to 'Priority insights' (TCO pipleine), Medium to 'Analyze and alert', Low to 'Store and search', Blocked are not sent to any pipeline.
+# application_rule - Define rules for matching applications to include in the policy configuration.
+# subsystem_rule - Define subsystem rules for matching applications to include in the policy configuration.
+# log_rules - Define the log severities to include in the policy configuration.
+# archive_retention - Define archive retention.
+##############################################################################################################
+
+variable "logs_policies" {
+  type = list(object({
+    logs_policy_name        = string
+    logs_policy_description = optional(string, null)
+    logs_policy_priority    = string
+    application_rule = optional(list(object({
+      name         = string
+      rule_type_id = string
+    })))
+    subsystem_rule = optional(list(object({
+      name         = string
+      rule_type_id = string
+    })))
+    log_rules = optional(list(object({
+      severities = list(string)
+    })))
+    archive_retention = optional(list(object({
+      id = string
+    })))
+  }))
+  description = "Configuration of Cloud Logs policies."
```

Comment:
Complex DA inputs need a helper markdown doc link. For example, see https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/2f888db2f038d0db6cffbb228cff53fd53652f04/solutions/security-enforced/variables.tf#L85

Reply:
It actually already exists in https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/main/solutions/instances/DA-types.md - you just need to extract the relevant info and add it to a markdown doc in this repo

Reply:
@kierramarie This is not addressed?